.travis.yml

```diff
@@ -11,6 +11,11 @@ matrix:
     - os: osx
       # Do not start osx build for PR
       if: type != pull_request
+      osx_image: xcode8
```

Comment:
@iketheadore will this impact our release builds?

Reply:
according to travis docs that's needed for `gcc6` which is needed to compile C testing binaries.